### PR TITLE
Feat/custom tabs content

### PR DESCRIPTION
**Pull request by robinandersson**
Tab Content sets height to 100%, which is not compatible with e.g. putting tabs in the Drawer component (scroll appears).

This PR: pass Content as optional prop

Suggestions for other possible solutions are appreciated.

### CODE REVIEW

File: src/Molecules/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -123,15 +124,17 @@ export const Tabs: ContainerComponent = ({
...
-            {c}
-          </Content>
+        const tabContentProps = {
+          id: `tabs-tabpanel-${i}`,
+          role: 'tabpanel',
+          'aria-labelledby': `tabs-tab-${i}`,
+          hidden: !isActive,
+        };
+
+        contents = TabContentWrapper ? (
```
Comment (almightybruce) [inline src/Molecules/Tabs/Tabs.tsx:134]: Looks good if you send something that can be JSX:ed, but will crash otherwise.. Perhaps some isElement check?

Comment (almightybruce) [reply src/Molecules/Tabs/Tabs.tsx:134]: Didn't see the React.ComponentType typing. You can probably ignore the previous comment

File: src/Molecules/Tabs/Tabs.types.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export type ContainerProps = {
   children: React.ReactNode;
   className?: string;
   height?: number;
+  TabContentWrapper?: React.ComponentType;
```
Comment (toblu) [inline src/Molecules/Tabs/Tabs.types.ts:26]: If you want to also support intrinsic JSX strings like (`"div"`, `"span"` etc), this should be `React.ElementType` https://flow.org/en/docs/react/types/#toc-react-elementtype

Comment (robinandersson) [reply src/Molecules/Tabs/Tabs.types.ts:26]: Do we want to allow such elements though? I'm not sure either or.

**Review by toblu — APPROVED** [review]

**Review by nn-davsko — APPROVED** [review]
Where did we land on this one? Do it or scrap it?

**Comment by robinandersson** [comment]
This PR is no longer needed or wanted.

To style the wrapper, use the approach introduced in #710

**Pull request closed without merging**